test/NuGet.Core.FuncTests/NuGet.XPlat.FuncTest/XPlatConfigTests.cs

```diff
@@ -20,35 +22,33 @@ public void ConfigPathsCommand_ListConfigPathsWithArgs_Success()
         {
             // Arrange & Act
             using var testInfo = new TestInfo("NuGet.Config");
-            {
-                var result = CommandRunner.Run(
-                      DotnetCli,
-                      Directory.GetCurrentDirectory(),
-                      $"{XplatDll} config paths {testInfo.WorkingPath}",
-                      waitForExit: true
-                      );
-
-                // Assert
-                DotnetCliUtil.VerifyResultSuccess(result, Path.Combine(testInfo.WorkingPath.Path, "NuGet.Config"));
-            }
+        
+            var result = CommandRunner.Run(
+                DotnetCli,
+                Directory.GetCurrentDirectory(),
+                $"{XplatDll} config paths {testInfo.WorkingPath}",
+                waitForExit: true
+                );
```

Comment:
Can you please fix this whole source file to not have the trailing parenthesis and semi-colon on a new line?  I see a lot of instances of it.
```suggestion
                waitForExit: true);
```